api_tests/regtest/docker-compose.yml

```diff
@@ -1,25 +1,24 @@
-version: '3'
+version: "3"
 services:
-  eth:
-    image: parity/parity:v2.5.0
-    command: --config dev --jsonrpc-apis=all --unsafe-expose --tracing=on --logging=debug,ethcore-miner=trace,miner=trace,rpc=trace,tokio_core=warn,tokio_reactor=warn --jsonrpc-cors="all"
-    ports:
-      - "8545:8545"
+    eth:
```

Comment:
Unfortunately, this would require renaming all the folders of our e2e tests.

In the long run, I'd like to get rid of the `harness.ts` in general and resort to `testcontainers` so that the actual __test__ in the files are the entrypoints again and don't depend on external state. That would simplify a lot of things.